tests/unit/test_clidocs.py

```diff
@@ -406,6 +407,32 @@ def test_includes_global_args_ref_in_html_options(self):
             "global parameters", rendered
         )
 
+    def test_includes_streaming_blob_options(self):
+        help_command = self.create_help_command()
+        blob_shape = Shape('blob_shape', {'type': 'blob'})
+        blob_shape.serialization = {'streaming': True}
+        blob_arg = CustomArgument('blob_arg', argument_model=blob_shape)
+        help_command.arg_table = {'blob_arg': blob_arg}
+        operation_handler = OperationDocumentEventHandler(help_command)
+        operation_handler.doc_option(arg_name='blob_arg',
+                                     help_command=help_command)
+        rendered = help_command.doc.getvalue().decode('utf-8')
+        self.assertIn('streaming blob', rendered)
+
+    def test_streaming_blob_comes_after_docstring(self):
+        help_command = self.create_help_command()
+        blob_shape = Shape('blob_shape', {'type': 'blob'})
+        blob_shape.serialization = {'streaming': True}
+        blob_arg = CustomArgument(name='blob_arg',
+                                  argument_model=blob_shape,
+                                  help_text='FooBar')
+        help_command.arg_table = {'blob_arg': blob_arg}
+        operation_handler = OperationDocumentEventHandler(help_command)
+        operation_handler.doc_option(arg_name='blob_arg',
+                                     help_command=help_command)
+        rendered = help_command.doc.getvalue().decode('utf-8')
+        self.assertRegex(rendered, 'FooBar[\s\S]*streaming blob')
```

Comment:
Let's make sure to prefix the regex string with an `r` (e.g. `r'FooBar[\s\S]*streaming blob'`) to make it a raw string. The reason I'm suggesting it is that if it the string is not prefixed with `r`, Python will throw deprecation warnings about invalid escape sequences being used in the literal string and this can make the pytest output noisy.